Automattic/Gravatar-SDK-Android
# Integrate Binary Compatibility Validator

Closes #179 

### Description

This PR adds the Binary Compatibility Validator plugin that will help us ensure backward compatibility with each release. 

There's an extra step added to the `.pipeline.yml` file that will run `./gradlew apiCheck` command so any changes to the API will have to be reviewed and approved.

### Testing Steps
- Review the public API, maybe there are some internal APIs that we want to exclude from the check that I'm not aware of. 
- CI step test to make sure it's being run when it should
